apps/server/src/modules/room/domain/do/room.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { EntityId } from '@shared/domain/types';
+import { Room, RoomProps } from './room.do';
+
+describe('Room', () => {
+	let room: Room;
+	const roomId: EntityId = 'roomId';
+	const roomProps: RoomProps = {
+		id: roomId,
+		name: 'Conference Room',
+		color: 'blue',
+		startDate: new Date('2024-01-01'),
+		untilDate: new Date('2024-12-31'),
+	};
+
+	beforeEach(() => {
+		room = new Room(roomProps);
+	});
+
+	it('should get and set name', () => {
+		expect(room.name).toBe('Conference Room');
+		room.name = 'Meeting Room';
```

Comment:
not sure how I feel about these tests, somehow feels like we are testing the basic functionality of nest here, feels unnecessary to me